04_numeronym/brnbke/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"unicode/utf8"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(vals ...string) []string {
+	var result = make([]string, 0, len(vals))
+	for _, val := range vals {
+		l := utf8.RuneCountInString(val)
+		if l > 3 {
+			s := []rune(val)[0]   // starting character
+			e := []rune(val)[l-1] // ending character
```

Comment:
a suggestion not required to implement, since you are already importing `unicode/utf8`: use `utf8.DecodeRuneInString()` and `utf8.DecodeLastRuneInString()`: https://golang.org/pkg/unicode/utf8/

Reply:
Implemented, it's considerably more efficient to use this also if my speed test was right